src/client/debugger/types.ts

```diff
@@ -33,3 +33,33 @@ export interface DebuggingDelegate {
      */
     willSendRequest(request: DebugProtocol.Request): Promise<void>;
 }
+
+export interface dumpCellResponse {
```

Comment:
Should these start with a capital letter or an `I`? I don't know if you have naming standards for these things in your repo

Reply:
We do, thanks for pointing it out